client/state/comments/actions.js

```diff
@@ -109,54 +56,29 @@ function commentsRequestFailure( dispatch, siteId, postId, requestId, err ) {
  * @param {Number} siteId site identifier
  * @param {Number} postId post identifier
  * @param {String} status status filter. Defaults to approved posts
+ * @param {Date}   before will retrieve comments before this date
```

Comment:
With `before`, will we query from this date to the current date? or does this mean from this date to before this date into the past?

It might be helpful to note this in the JSDoc